notifier.go

```diff
@@ -33,23 +33,34 @@ func New(rawData ...interface{}) *Notifier {
 // Notify sends an error to Bugsnag. Any rawData you pass here will be sent to
 // Bugsnag after being converted to JSON. e.g. bugsnag.SeverityError, bugsnag.Context,
 // or bugsnag.MetaData.
-func (notifier *Notifier) Notify(err error, rawData ...interface{}) (e error) {
-	config := notifier.Config
-	return notifier.NotifySync(err, config.Synchronous, rawData...)
+func (notifier *Notifier) Notify(rawData ...interface{}) (e error) {
+	return notifier.NotifySync(append(rawData, notifier.Config.Synchronous)...)
```

Comment:
The order of the raw data variables have changed, leading to a different precedence for deciding if the report should be sent synchronously. Before it would use the config.Synchronous first, but then it could be overwritten by a config object or boolean in the raw data. Now, it doesn't matter what the raw data is, it will always use the config.Synchronous variable since its the last parameter.

Reply:
Note to self: fix this by changing
```go
NotifySync(append(rawData, notifier.Config.Synchronous)...)
```
into
```go
NotifySync(append(arrayContainingConfigSynchronous, allElementsOfRawData)...)
```